### PR TITLE
Add docker tasks

### PR DESCRIPTION
**Pull request by jitekuma**
Add docker  and docker  compose vsts tasks.

### CODE REVIEW

File: Tasks/DockerCompose/dockerCompose.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import * as tl from "vsts-task-lib/task";
+import DockerComposeConnection from "./dockerComposeConnection";
+
+// Change to any specified working directory
+tl.cd(tl.getInput("cwd"));
+
+var dockerComposeFile = tl.getInput("dockerComposeFile", true);
+var nopIfNoDockerComposeFile = tl.getBoolInput("nopIfNoDockerComposeFile");
+if (nopIfNoDockerComposeFile && !tl.globFirst(dockerComposeFile)) {
+    console.log("No Docker Compose file matching " + dockerComposeFile + " was found.");
```
Comment (omeshp) [inline Tasks/DockerCompose/dockerCompose.ts]: add these strings for localization.

File: Tasks/DockerCompose/task.json [hunk 2]
```diff
@@ -0,0 +1,272 @@
+{
+    "id": "6975E2D1-96D3-4AFC-8A41-498B5D34EA19",
+    "name": "DockerCompose",
+    "friendlyName": "Docker Compose",
+    "description": "Build, push or run multi-container Docker applications",
+    "helpMarkDown": "[More Information](http://aka.ms/dockertoolsforvsts)",
+    "category": "Build",
+    "visibility": [
```
Comment (omeshp) [inline Tasks/DockerCompose/task.json]: visibility is deprecated, we should not add it anymore

File: Tasks/DockerCompose/task.json [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+            "label": "Abort on Container Exit",
+            "defaultValue": "true",
+            "visibleRule": "action = Run services && detached == false",
+            "helpMarkDown": "Stop all containers when any container exits."
+        },
+        {
+            "name": "imageDigestComposeFile",
+            "type": "filePath",
+            "label": "Image Digest Compose File",
+            "defaultValue": "$(Build.StagingDirectory)/docker-compose.images.yml",
```
Comment (omeshp) [inline Tasks/DockerCompose/task.json]: we should use default variables which are available both in build and release.

**Comment by jitekuma** [comment]
Creating new pull request. Abandoned this

**Pull request closed without merging**